packages/kotti-ui/source/kotti-field/constants.ts

```diff
@@ -1,45 +1,4 @@
-import { isYocoIcon } from '@3yourmind/yoco'
-
-import { KottiField } from './types'
-
-export const KOTTI_FIELD_INHERITABLE_PROPS = {
-	hideClear: { default: false, type: Boolean },
-	hideValidation: { default: false, type: Boolean },
-	isDisabled: { default: false, type: Boolean },
-	isLoading: { default: false, type: Boolean },
-	size: {
-		default: null,
```

Comment:
medium

Reply:
bring back the null to distinguish the size passed down from context  as default null